<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformList</class>
 <widget class="QWidget" name="PerformList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>添加一项</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>13</number>
     </property>
     <column>
      <property name="text">
       <string>某立绘</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>某表情</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X坐标</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y坐标</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>缩放比</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>持续时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>是否显示于左下角</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>动画效果</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>持续时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>动画幅度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
